parsers/US_NEISO.py

```diff
@@ -69,7 +77,9 @@ def get_json_data(
     return raw_data
 
 
-def production_data_processer(raw_data, logger: Logger) -> list:
+def production_data_processer(
+    zone_key: ZoneKey, raw_data: dict, logger: Logger
+) -> ProductionBreakdownList:
```

Comment:
```suggestion
def production_data_processer(
    zone_key: ZoneKey, raw_data: Union[dict, list], logger: Logger
) -> ProductionBreakdownList:
```
Json can both be a dict and list.